test/decompiler/reference/levels/rolling/rolling-obs_REF.gc

```diff
@@ -872,27 +871,40 @@
   )
 
 ;; definition for function race-time-save
-(defun race-time-save ((arg0 race-time) (arg1 int) (arg2 task-control))
-  (dotimes (s3-0 5)
-    (save-reminder arg2 (-> arg0 digit s3-0) (+ arg1 s3-0))
-    )
-  #f
+;; WARN: Type Propagation failed: Failed type prop at op 6 ((set! v1 (l.wu (+ a0 -4)))): Could not get type of load: (set! v1 (l.wu (+ a0 -4))). 
+;; WARN: Type Propagation failed: Type analysis failed
+;; WARN: Function may read a register that is not set: a2
+(defun race-time-save ((a0-0 race-time) (a1-0 task-control))
```

Comment:
this one looks kinda broken. If we want all these patches, I wonder if we should just switch to decompiling the PAL version. Having all these combined versions is kinda confusing

Reply:
Yeah I wouldn't mind changing the ref tests to use PAL or something after that.